### ISSUE TITLE
Introduce callouts in default theme

### ISSUE THREAD

**Issue by legoming** [MacDownApp/macdown#1074]
I defined some callouts by referring to Google's markdown style, is this possible to introduce them into default theme?

<img src="https://raw.githubusercontent.com/legoming/macdown/master/assets/callout.png" style="width:300px">

**Comment by FranklinYu** [comment]
Not really interested in it since it’s not portable, but PR welcome. (Just make it opt-in.)

**Comment by legoming** [comment]
Yes. The callout format is lost when I open it with Typora who doesn’t support customized html tag.
But is really useful when creating documents.
